avm/res/operations-management/solution/main.bicep

```diff
@@ -2,7 +2,16 @@ metadata name = 'Operations Management Solutions'
 metadata description = 'This module deploys an Operations Management Solution.'
 metadata owner = 'Azure/module-maintainers'
 
-@description('Required. Name of the solution. For Microsoft published gallery solution the target solution resource name will be composed as `{name}({logAnalyticsWorkspaceName})`.')
+@metadata({
+  example: '''
```

Comment:
Why not add the example to the description so that it's part of the readme?

Reply:
Nevermind, I see it is also in the description - which raises the opposite question: Why? 😄

Reply:
I tried the example metadata but I couldn't locate the example in the readme. Let me check again, maybe I overlooked something...

Reply:
That's my point. It's not a supported property 😄 I'd just keep the example in the description and remove the other

Reply:
Now it should be fine! :)